src/ape/pytest/runners.py

```diff
@@ -161,6 +161,19 @@ def pytest_sessionstart(self):
         if warnings and not self.config_wrapper.disable_warnings:
             reporter.stats["warnings"] = warnings
 
+    @pytest.hookimpl(tryfirst=True, hookwrapper=True)
+    def pytest_runtest_call(self, item):
+        marker = item.get_closest_marker("use_network")
+        if marker:
```

Comment:
Can also use walrus operator


```suggestion
        if marker := item.get_closest_marker("use_network"):
```